test/test_torch.py

```diff
@@ -6704,9 +6704,9 @@ def test_parsing_intlist(self):
 
         # fail parse with > 1 element variables
         self.assertRaises(TypeError, lambda: torch.ones(torch.tensor(3, 3)))
-        self.assertRaises(TypeError, lambda: torch.ones(torch.tensor(3, 3)))
-        self.assertRaises(TypeError, lambda: torch.ones(np.array(3, 3)))
+        self.assertRaises(TypeError, lambda: torch.ones((torch.tensor(3, 3),)))
```

Comment:
Wait this is semantically not the same thing! What suggested this change?

Reply:
I think we had talked about this before in this PR. I did it manually.

But it makes sense, right? Because, otherwise, there are duplicate asserts.

Reply:
@bryant1410 I think you likely meant to deduplicate lines 6706 and 6707 NOT 6707 and 6708

Reply:
The diff is confusing but the end result is fine IMHO.

Before this change, there were 2 pairs of duplicate lines. They, in turn, before a recent [BE] PR merge, had been originally syntactically different but semantically the same because they had redundant parentheses which the mentioned PR removed.

So, my change is not to remove the duplicates but to fix what I interpret the original intention was, which is to assert that the function call works both with and without a single-element tuple (for each pair of duplicates).

Reply:
Hey!
I would have two things to say:
- We should NOT do semantic changes in pyupgrade/lint PRs. If we want to do this, this should be a separate PR.
- Note that 6700 above basically tests this case already. So I don't think we need to add this.

Given this, in this PR, we should just remove these lines.